editor/editor_spin_slider.cpp

```diff
@@ -181,6 +182,39 @@ void EditorSpinSlider::_grabber_gui_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+void EditorSpinSlider::_value_input_gui_input(const Ref<InputEvent> &p_event) {
+	Ref<InputEventKey> k = p_event;
+	if (k.is_valid() && k->is_pressed()) {
+		uint32_t code = k->get_keycode();
+		double step = get_step();
+
+#ifdef APPLE_STYLE_KEYS
+		if (k->get_command()) {
+#else
+		if (k->get_control()) {
+#endif
```

Comment:
Is this #if required? Can't you just use `k->get_command()` since on Non-Apple systems, Command and Control are the same thing.

Reply:
I followed the code that was in other nodes on the assumption that it was required there. See `line_edit.cpp` and `text_edit.cpp`.